### PR TITLE
[WIP] Added email subject to incident model and form

### PR DESCRIPTION
**Pull request by bmsauer**
I added an extra field for email subject lines for incidents.  It stores it in the incident table.  This is somewhat related to https://github.com/cachethq/Cachet/issues/1009 .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 node_modules
 phpunit.xml
 vendor
+*~
```
Comment (GrahamCampbell) [inline .gitignore]: please erert that change

File: resources/views/dashboard/incidents/add.blade.php [hunk 2]
```diff
@@ -115,6 +115,10 @@
                                 {{ trans('forms.incidents.notify_subscribers') }}
                             </label>
                         </div>
+			<div class="form-group">
```
Comment (GrahamCampbell) [inline resources/views/dashboard/incidents/add.blade.php]: Please use spaces for indentation.

**Comment by GrahamCampbell** [comment]
Please fix the failing travis tests.

File: database/migrations/2016_05_26_131224_add_incident_email_subject_column.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('incidents', function ($table) {
+            $table->string('email_subject');
```
Comment (jbrooksuk) [inline database/migrations/2016_05_26_131224_add_incident_email_subject_column.php]: This column should be added before the timestamps fields, but after `message`.

**Comment by jbrooksuk** [comment]
What is the use case for this?

**Comment by jbrooksuk** [comment]
In my opinion, we should make the subjects standardised, such as "New incident, `$title`".

File: .gitignore (hunk 1 above)
Comment (bmsauer) [inline .gitignore]: This is to ignore out emacs autosave files.

File: database/migrations/2016_05_26_131224_add_incident_email_subject_column.php (hunk 3 above)
Comment (bmsauer) [inline database/migrations/2016_05_26_131224_add_incident_email_subject_column.php]: I'm not sure I understand what you mean, can you explain more?

**Comment by bmsauer** [comment]
I'm working on doing the tests now.  As for the use case, we just want to be specific for our customers, as some incidents within components still only affect a subset of the subscribers.

File: .gitignore (hunk 1 above)
Comment (GrahamCampbell) [inline .gitignore]: Please ad this to your global git ignore file, and revert this change.

**Comment by GrahamCampbell** [comment]
Thanks, but this isn't something we want to add to Cachet right now. Do check out the recent changes we've just merged though. ;)
